### ISSUE TITLE
No Audio From Live TV

### ISSUE THREAD

**Issue by Jrbesse** [jellyfin/Swiftfin#658]
### Describe the bug

1. Go to Settings -> Experimental -> check Live TV (Alpha)
2. Play any channel

Video will show as expected but there is no audio, regardless if the channel is is providing stereo or 5.1 sound.
This is using a HD HomeRun Flex 4k.
I am getting audio without issue on the Jellyfin mobile ios app, version 1.5.0 (5.5.0.0), Expo Version 43.0.0

Please let me know what logs or additional data would be useful.

### Application version

1.0.0(79)

### Where did you install the app from?

App Store

### Device information

iPhone 13 Pro

### OS version

iOS 16.1.2

### Jellyfin server version

10.8.8

**Comment by zapotocnylubos** [comment]
same here

**Comment by brutuscat** [comment]
Same issue on iPhone 14

**Comment by brutuscat** [comment]
Yeah, solved as soon as I selected "Live TV Native Player" option in the experimental section.
